pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/fraser/ufoer/Barnetillegg.kt

```diff
@@ -0,0 +1,708 @@
+        val faarUtbetaltBarnetilleggFellesBarn: Expression<Boolean>,
+        val harFradragFellesbarn: Expression<Boolean>,
+        val fribeloepFellesbarn: Expression<Kroner>,
+        val grunnbeloep: Expression<Kroner>,
+        val harBarnetilleggSaerkullsbarn: Expression<Boolean>,
+        val inntektAnnenForelderFellesbarn: Expression<Kroner>,
+        val inntektBruktiAvkortningFellesbarn: Expression<Kroner>,
+        val harJusteringsbeloepFellesbarn: Expression<Boolean>,
+        val sivilstand: Expression<Sivilstand>
+
+    ) : OutlinePhrase<LangBokmalNynorskEnglish>() {
+        override fun OutlineOnlyScope<LangBokmalNynorskEnglish, Unit>.template() {
+            val inntektLavereEnnFribeloep = inntektBruktiAvkortningFellesbarn.lessThanOrEqual(fribeloepFellesbarn)
+            val grunnbeloep = grunnbeloep.format()
+
+            showIf(not(harBarnetilleggSaerkullsbarn)) {
+                paragraph {
+                    textExpr(
+                        Bokmal to "Inntekten din er ".expr() + inntektBruktiAvkortningFellesbarn.format() + " kroner og inntekten til ".expr(),
+                        Nynorsk to "Inntekta di er ".expr() + inntektBruktiAvkortningFellesbarn.format() + " kroner, og inntekta til ".expr(),
+                        English to "Your income is NOK ".expr() + inntektBruktiAvkortningFellesbarn.format() + " and your ".expr()
+                    )
+
+                    includePhrase(Felles.SivilstandEPSBestemtForm(sivilstand))
+
+                    textExpr(
+                        Bokmal to " din er ".expr() + inntektAnnenForelderFellesbarn.format() + " kroner.".expr(),
+                        Nynorsk to " din er ".expr() + inntektAnnenForelderFellesbarn.format() + " kroner.".expr(),
+                        English to "'s income is NOK ".expr() + inntektAnnenForelderFellesbarn.format() + "."
+                    )
+
+                    showIf(harBeloepFratrukketAnnenForelder) {
+                        textExpr(
+                            Bokmal to " Folketrygdens grunnbeløp på inntil ".expr() + grunnbeloep + " kroner er holdt utenfor inntekten til ".expr(),
+                            Nynorsk to " Grunnbeløpet i folketrygda på inntil ".expr() + grunnbeloep + " kroner er halde utanfor inntekta til ".expr(),
+                            English to " The national insurance basic amount of up to NOK ".expr() + grunnbeloep + " has not been included in your ".expr()
+                        )
+                        includePhrase(Felles.SivilstandEPSBestemtForm(sivilstand))
+
+                        text(Bokmal to " din.", Nynorsk to " din.", English to "'s income.")
+                    }
+
+
+
+                    showIf(harJusteringsbeloepFellesbarn) {
+                        showIf(faarUtbetaltBarnetilleggFellesBarn) {
+                            textExpr(
+                                Bokmal to " Til sammen er inntektene ".expr() +
```

Comment:
Kanskje ikke nøvendig å endre på nå, men noe vi kan tenke på. Burde vi hatt automatikk i at etterfølgende text/textExpr som standard får mellomrom mellom seg? (Som kan endres gjennom en optional parameter?)

Reply:
Det blir knotete. Spesielt om du scroller litt opp til "text(Bokmal to " din.", Nynorsk to " din.", English to "'s income.")"